README.md

```diff
@@ -87,7 +87,7 @@ Open from disk:
 
 Get key/value:
 ```go
-  val, exists, err = fst.Get([]byte("dog"))
+  exists, val, err = fst.Get([]byte("dog"))
```

Comment:
bike shed'ing, but I kinda liked the original val, exists, err ordering as it echoed the val, exists ordering that you experience from the built-in golang map[] lookups.